webapp-django/crashstats/documentation/templates/documentation/supersearch/home.html

```diff
@@ -0,0 +1,389 @@
+                        <td>Default operator. The field contains at least once the given substring. </td>
+                    </tr>
+                    <tr>
+                        <td>contains</td>
+                        <td><code>~</code></td>
+                        <td>string</td>
+                        <td>The field contains at least once the given substring.</td>
+                    </tr>
+                    <tr>
+                        <td>is exactly</td>
+                        <td><code>=</code></td>
+                        <td>string</td>
+                        <td>The field is exactly the given substring.</td>
+                    </tr>
+                    <tr>
+                        <td>starts with</td>
+                        <td><code>$</code></td>
+                        <td>string</td>
+                        <td>The field starts with the given substring.</td>
+                    </tr>
+                    <tr>
+                        <td>ends with</td>
+                        <td><code>^</code></td>
+                        <td>string</td>
+                        <td>The field ends with the given substring.</td>
+                    </tr>
+                    <tr>
+                        <td>is null</td>
+                        <td><code>__null__</code></td>
+                        <td>string, flag</td>
+                        <td>The field is missing or has a null value.</td>
+                    </tr>
+                    <tr>
+                        <td>greater than</td>
+                        <td><code>&gt;</code></td>
+                        <td>date, number</td>
+                        <td>The field is greater than the given date or number.</td>
+                    </tr>
+                    <tr>
+                        <td>greater than or equal</td>
+                        <td><code>&gt;=</code></td>
+                        <td>date, number</td>
+                        <td>The field is greater than or equal the given date or number.</td>
+                    </tr>
+                    <tr>
+                        <td>lower than</td>
+                        <td><code>&lt;</code></td>
+                        <td>date, number</td>
+                        <td>The field is lower than the given date or number.</td>
+                    </tr>
+                    <tr>
+                        <td>lower than or equal</td>
+                        <td><code>&lt;=</code></td>
+                        <td>date, number</td>
+                        <td>The field is lower than or equal the given date or number.</td>
+                    </tr>
+                    <tr>
+                        <td>is true</td>
+                        <td><code>__true__</code></td>
+                        <td>boolean</td>
+                        <td>The field has a value that is truthy.</td>
+                    </tr>
+                </tbody>
+            </table>
+
+            <h2>Data types</h2>
```

Comment:
Move this awesome part above the list above. Then the list of all operators doesn't really need a "Data types" column. That way you don't need to maintain two lists of operator-to-data-types tables.